ivy_tests/test_ivy/test_misc/test_backend_utils/test_backend_handler.py

```diff
@@ -88,32 +88,33 @@ def test_current_backend(backend, array_type):
 
 
 @pytest.mark.parametrize(
-    "middle_backend,end_backend", [(a, b) for a in backends for b in backends if a != b]
+    "middle_backend,end_backend",
+    [(a, b) for a in backends for b in backends if (a != b and "mxnet" not in [a, b])],
 )
 def test_dynamic_backend_all_combos(middle_backend, end_backend):
     # create an ivy array, container and native container
     a = ivy.array([1, 2, 3])
     b = ivy.array([4, 5, 6])
     ivy_cont = ivy.Container({"w": a, "b": b})
-    nativ_cont = ivy.Container(
```

Comment:
are we skipping this test now (with native array containers)?

Reply:
I gave this a lot of thought, and I couldn't see how the dynamic backend switch should change a non-ivy native variable to the new backend, this is a very invasive behaviour and shouldn't be expected, instead I added a new test case for ivy-wrapped variable container

Reply:
yeah sounds good!